storage/tianmu/core/item_tianmu_field.cpp

```diff
@@ -118,7 +123,11 @@ void Item_tianmufield::FeedValue() {
   switch (tianmu_type.valtype) {
     case DataType::ValueType::VT_FIXED:
       if (tianmu_type.IsInt())
-        ((Item_int *)ivalue)->value = buf->Get64();
+        if (unsigned_flag) {
+          ((Item_float *)ivalue)->value = static_cast<ulonglong>(buf->Get64());
+        } else {
+          ((Item_int *)ivalue)->value = buf->Get64();
```

Comment:
`((Item_int *)ivalue)` using c++ style type-cast. Thxs.

Reply:
fixed it.